src/lib.rs

```diff
@@ -581,6 +581,17 @@ impl<T> Either<T, T> {
     pub fn into_inner(self) -> T {
         either!(self, inner => inner)
     }
+    
+    /// Maps `f` over the contained value and returns the result in the
+    /// corresponding variant.
```

Comment:
I swear there is nothing to complain about, but I'll just mention that active form is the better for method docs "Map f over the contained value and return .. " agree?  I can see that not 100% of the existing methods in the crate agree with this.